test_runner/src/main/kotlin/ftl/run/DumpShards.kt

```diff
@@ -12,31 +13,31 @@ import ftl.util.obfuscatePrettyPrinter
 import java.nio.file.Files
 import java.nio.file.Paths
 
-suspend fun dumpShards(
-    args: AndroidArgs,
+suspend fun AndroidArgs.dumpShards(
+    @VisibleForTesting
```

Comment:
`@VisibleForTesting` is used for field, class, method visibility, not for a method parameter, which do not have any visibility. We should not use it

Reply:
For me, it's the only convention for marking why something is exposed but is not used in production code.